docs/static/releasenotes.asciidoc

```diff
@@ -14,6 +15,34 @@ See also:
 * <<release-notes-xls>>
 endif::include-xpack[]
 
+[[logstash-6-2-0]]
+=== Logstash 6.2.0 Release Notes
+
+* Added support to protect sensitive settings and configuration in a https://www.elastic.co/guide/en/logstash/current/keystore.html[keystore]
+* https://www.elastic.co/guide/en/logstash/current/multiple-pipelines.html[Multiple pipelines] is now the default configuration (applicable for RPM/DEB/Docker installations)
```

Comment:
+* Set default configuration for RPM/DEB/Docker installations to https://www.elastic.co/guide/en/logstash/current/multiple-pipelines.html[Multiple pipelines]

Reply:
done with minor variant:  to *use* Multiple pipelines.